core/renderers/common/constants.js

```diff
@@ -571,6 +569,60 @@ Blockly.blockRendering.ConstantProvider.prototype.refreshTheme = function(
   }
 };
 
+/**
+ * Set dynamic constants that depent on other values or theme properties.
+ * @param {!Blockly.Theme} theme The current workspace theme.
+ * @protected
+ */
+Blockly.blockRendering.ConstantProvider.prototype.setDynamicConstants_ =
+    function(theme) {
+    /* eslint-disable indent */
```

Comment:
You need an `eslint-enable` line for each of these, or it applies to the rest of the file.